.gitignore

```diff
@@ -17,3 +17,4 @@ docs/public
 packs/*.tgz
 package-lock.json
 .nvmrc
+storybook-static
```

Comment:
Well, I noticed that this line is missing while testing this PR, so some relation is present =)